samples/nrf9160/lwm2m_client/src/events/include/sensor_event.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SENSOR_EVENT_H__
+#define SENSOR_EVENT_H__
+
+#include <zephyr.h>
+#include <net/lwm2m.h>
+#include <event_manager.h>
+#include <drivers/sensor.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum sensor_type {
+	light_sensor,
```

Comment:
nit: enum values are typically written in all UPPER_CASE, should be a simple search and replace.

Reply:
Fixed